### DISCUSSION TITLE
GOG Games Directory

### DISCUSSION THREAD

**Discussion by synthsncats** [tkashkin/GameHub#477] [Q&A] [answered]
What does Gamehub do to my GOG Games directory? Does it merge it with it's own directory? If so, where is it?

**Comment by tkashkin** [comment, marked as answer]
It depends on what do you mean by GOG Games Directory.

## If it's the option in Settings (`~/Games/GOG` by default):

That's where GameHub installs games from GOG.

## If it's the literal `~/GOG Games` directory:

It is the default installation directory for native GOG games.
Currently GameHub does nothing with it.

`refactoring` branch supports multiple game directories per game source and has `~/GOG Games` in the list by default.
I'll try to implement automatic import of games that were not installed by GameHub.
